mat64/triangular.go

```diff
@@ -181,7 +184,11 @@ func (t *TriDense) isZero() bool {
 // reuseAS resizes a zero receiver to an n×n triangular matrix with the given
 // orientation. If the receiver is non-zero, reuseAs checks that the receiver
 // is the correct size and orientation.
-func (t *TriDense) reuseAs(n int, ul blas.Uplo) {
+func (t *TriDense) reuseAs(n int, upper bool) {
```

Comment:
Not directly, no, just that it seems like one should be able to say
````
t.reuseAs(a.Triangular()) 
````
rather than have to do a switch on upper and uplo every time.